src/utils/processWhitelist.spec.js

```diff
@@ -83,7 +83,7 @@ describe('processWhitelist function', () => {
     const cb = jest.fn()
 
     // When
-    const { called } = processWhitelist(rows, cb)
+    const { called } = processWhitelist({ rows, decimals: 3 }, cb)
```

Comment:
Could you add some test cases for ignoring invalid decimals, and invalid min/max combinations?

Reply:
done